src/legacy/core_plugins/expressions/public/np_ready/public/expression_renderer.tsx

```diff
@@ -17,58 +17,117 @@
  * under the License.
  */
 
-import { useRef, useEffect } from 'react';
+import { useRef, useEffect, useState } from 'react';
 import React from 'react';
-import { ExpressionAST, IExpressionLoaderParams, IInterpreterResult } from './types';
-import { IExpressionLoader, ExpressionLoader } from './loader';
+import classNames from 'classnames';
+import { EuiLoadingChart, EuiProgress } from '@elastic/eui';
+import { ExpressionAST, IExpressionLoaderParams, IInterpreterErrorResult } from './types';
+import { ExpressionLoader } from './loader';
 
 // Accept all options of the runner as props except for the
 // dom element which is provided by the component itself
 export interface ExpressionRendererProps extends IExpressionLoaderParams {
   className: string;
```

Comment:
I wouldn't allow users of this component modify its CSS, as it is prone to unexpected bugs.

Instead a user of this component could wrap it in a `<div>` and style that however they want.

```tsx
<div className={className}>
  <ExpressionRenderer expression="..." />
</div>
```

Or, `<ExpressionRenderer>` could provide props that modify its styling behaviour.

```tsx
<ExpressionRenderer />
<ExpressionRenderer block />
<ExpressionRenderer flex />
```

Reply:
Got it, fixed.